frame/babe/src/mock.rs

```diff
@@ -70,19 +81,40 @@ impl frame_system::Trait for Test {
 
 impl_opaque_keys! {
 	pub struct MockSessionKeys {
-		pub dummy: UintAuthorityId,
+		pub babe_authority: super::Module<Test>,
 	}
 }
 
 impl pallet_session::Trait for Test {
 	type Event = ();
 	type ValidatorId = <Self as frame_system::Trait>::AccountId;
 	type ShouldEndSession = Babe;
-	type SessionHandler = (Babe,Babe,);
-	type SessionManager = ();
+	type SessionHandler = (Babe,);
+	type SessionManager = pallet_session::historical::NoteHistoricalRoot<Self, Staking>;
 	type ValidatorIdOf = ();
```

Comment:
this one changed upstream also
```suggestion
	type ValidatorIdOf = pallet_staking::StashOf<Self>;
```